### DISCUSSION TITLE
How to enable wrapped tabs? (or alternatively vertical tabs)

### DISCUSSION THREAD

**Discussion by rrmistry** [eclipse-theia/theia#13974] [Q&A] [answered]
How can I enable wrapped editor tabs like in VS Code.

The current vscode setting is below:

![Screenshot_20240731-171017_Edge.png](https://github.com/user-attachments/assets/fc43404e-eaf0-4bd6-8eb7-57690949904b)

I could not find the corresponding setting in Theia

An alternative is vertical tabs. There is a long running issue with VS Code https://github.com/microsoft/vscode/issues/108264 that, if resolved in Theia, would convert a lot of users

**Comment by msujew** [comment, marked as answer]
Hey @rrmistry,

we have an open issue for this feature over at https://github.com/eclipse-theia/theia/issues/10383. There was an attempt to add this in https://github.com/eclipse-theia/theia/pull/10499, but that led to a lot of display issues. I don't think anyone has attempted to support this since then. 

As for https://github.com/microsoft/vscode/issues/108264: Do you want to create a feature proposal in this repo for this? Otherwise it's hard to keep track of this.
